baseline/client/spatial-orientation/spatial-orientation.js

```diff
@@ -13,20 +15,60 @@ import test_instruction_html from "./frag/test_instruction.html";
 import completion_html from "./frag/completion.html";
 
 export class SpatialOrientation {
+    constructor(setNum) {
+        if (Number.isInteger(setNum) && setNum > 0) {
+            this.setNum = setNum
+        } else {
+            throw new Error("setNum must be a strictly positive integer");
+        }
+        // get test stimulus
+        const setStim = this.constructor.stimulus.pre.main[String(setNum)];
```

Comment:
I hate to say this now (after previously saying it wasn't critical before), but our conversation today about distinguishing pre/post makes me realize that we really should go back and change the stimulus file to eliminate the pre/post keys and just do everything off of set number. Otherwise we're going to have a hardwired pre/post boundary in here just because it's unnecessarily set in the stimulus file. I can make the changes to the stimulus file if you think it's going to be a pain - just let me know and I'll submit a PR for the Floozutter sot-task branch.

Reply:
I'll make the update, no worries. I already have some Python lying around made for the spreadsheet-to-JSON conversion and I can just repurpose that.